auto-generated/cluster/stone-prd-rh01/tenants/rhtap-build-tenant/v1_serviceaccount_appstudio-pipeline.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+imagePullSecrets:
+- name: registry-pull-secret
+kind: ServiceAccount
+metadata:
+  name: appstudio-pipeline
+  namespace: rhtap-build-tenant
+secrets:
+- name: registry-pull-secret
```

Comment:
> Note that the list of secrets is not exhaustive - the secrets declared
here will be appended to those managed by Konflux.

Tested with `oc apply` locally. I hope it works the same way with ArgoCD.

Backed up the rhtap-build-tenant/appstudio-pipeline SA yaml just in case.

Reply:
FWIW it seems to work the same way for the build-templates-e2e SA https://github.com/redhat-appstudio/infra-deployments/blob/main/components/build-templates/production/e2e-serviceaccount.yaml

```bash
$ oc get -n build-templates-e2e sa appstudio-pipeline -o yaml | yq .secrets
- name: quay-push-secret
- name: appstudio-pipeline-dockercfg-9mkwp
- name: registry-redhat-io-pull-secret
- name: devfile-sample-python-basic-koxy
- name: retrodep-cqec
- name: pip-e2e-test-ubar
- name: fbc-sample-repo-pp2d
- name: nodejs-no-dockerfile-6q5l
- name: maven-hello-world-jnbq
- name: devfile-sample-python-basic-xnvk
```